examples/09_svcmulti/CMakeLists.txt

```diff
@@ -0,0 +1,2 @@
+include_directories(${AREG_EXAMPLES}/09_service/)
```

Comment:
Sorry but I am not sure what this is for. Could you please help explain it a bit?

Reply:
Well, in `main.cpp` file for including header files, this method is used:

```c
#include "src/ServicingComponent.hpp"
```

which is not a path from `areg-sdk` root directory, therefore by using `include_directories()`, we gives CMake valid path to them. In this case `src/ServicingComponent.hpp`.

Reply:
Then should it be`${AREG_EXAMPLES}/09_svcmulti/` instead?

Reply:
Oh yes, sorry typo :)